src/Controls/samples/Controls.Sample.Gtk/SimpleSampleApp/SimpleSampleGtkApplication.cs

```diff
@@ -25,7 +25,6 @@ Widget OnTopContainerOverride(Widget nativePage)
 		{
 			var b = new Box(Orientation.Vertical, 0)
 			{
-				Fill = true,
 				Expand = true,
```

Comment:
we had an api change in GktSharp:
https://github.com/GtkSharp/GtkSharp/commit/afc7ab450b90b8cdde122a99eed690e29b5a1da7
so this is needed to use the latest version